tests/lib/rendererCoverageCollector.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * Utilities for collection of renderer processes coverage
+ *
+ * prepending and appending necessary sequence elements to the test definitions and
+ * for bootstraping the test application instance.
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+
+var fluid = require("gpii-universal"),
+    gpii = fluid.registerNamespace("gpii"),
+    ipcMain = require("electron").ipcMain,
+    BrowserWindow = require("electron").BrowserWindow;
+
+require("gpii-testem");
+
+
+
+fluid.defaults("gpii.tests.app.rendererCoverageServer", {
+    gradeNames:  "gpii.testem.coverage.express",
+    port: 7003,
+    distributeOptions: {
+        record: "%gpii-app/coverage",
+        target: "{that gpii.testem.coverage.receiver.middleware}.options.coverageDir"
+    },
+    listeners: {
+        "onCreate": { // XXX dev
+            funcName: "console.log",
+            args: ["ROUTER CREATED"]
+        }
+    }
+});
```

Comment:
So, here's where you would reconfigure the coverage client to hook into QUnit rather than Testem, using code like the following:

```
    components: {
        coverageRouter: {
            type: "gpii.testem.coverage.router",
            options: {
                            components: {
                                client: {
                                    options: {
                                        hookTestem: false,
                                        hookQUnit: true
                                    }
                                }
                     }
            }
        }
    }
```

Once you've done that, you should be able to load the coverage client and have it hook into the tests you're running.

Reply:
Currently we are not running separate tests in the renderer processes but instead make interactions with the different BrowserWindows and collect coverage for those